lib/openers/qpdfview-opener.js

```diff
@@ -0,0 +1,35 @@
+/** @babel */
+
+import _ from 'lodash'
+import Opener from '../opener'
+import { isPdfFile, isPsFile } from '../werkzeug'
+
+export default class QPdfViewOpener extends Opener {
+  executable = 'qpdfview'
+
+  async initialize () {
+    this.exists = false
+    try {
+      if (process.platform === 'linux') {
+        const { statusCode } = await latex.process.executeChildProcess(`${this.executable} --help`)
+        this.exists = statusCode === 0
+      }
+    } catch (e) {}
+  }
+
+  async open (filePath, texPath, lineNumber) {
+    const command = `${this.executable} --unique "${filePath}#src:${texPath}:${lineNumber}:0"`
+    await latex.process.executeChildProcess(command, { showError: true })
+  }
+
+  async canOpen (filePath) {
+    if (_.isUndefined(this.exists)) {
```

Comment:
Is there some special reason for using `_.isUndefined(this.exists)` instead of simply checking `this.exists === undefined`? Seems very unnecessary to retain a dependency on lodash for something like this.

---
For reference, my suggested replace above is exactly how `isUndefined` is [implemented](https://github.com/lodash/lodash/blob/4.16.4/lodash.js#L12138), a simple comparison to `undefined`.